include/nbl/system/IAsyncQueueDispatcher.h

```diff
@@ -90,6 +101,12 @@ class IAsyncQueueDispatcher : public IThreadHandler<CRTP, InternalStateType>, pu
         assert(req.ready);
     }
 
+protected:
+    bool process_request_predicate(const request_t& req)
```

Comment:
musing, should this be a const method, static method?

Also should the request parameter be a const reference or not?

Reply:
static no, i dont want to limit what user wants to do
const? maybe, doesnt really matter too much

Reply:
stays as is, resolved.